src/pages/api/getGPTDistractor.ts

```diff
@@ -0,0 +1,34 @@
+import { apiController } from '@utils/api'
+import { Env } from '@utils/getEnv'
+import { Configuration, OpenAIApi } from 'openai'
+
+export interface GetGPTDistractorsParams {
+  qStem: string
+  qLearningObjective: string
+}
+
+export interface GetGPTDistractorsResults {
+  distractors: string[]
+}
+
+export default apiController<GetGPTDistractorsParams, GetGPTDistractorsResults>(
+  async ({ qStem, qLearningObjective }) => {
+    const promptQuestion = `Create 3 distractors separated by a comma for the following question ${qStem}; its learning objective is ${qLearningObjective}, reply only with the distractors.`
+
+    const openAIKey = Env.OPEN_AI_KEY
+    const openAI = new OpenAIApi(
+      new Configuration({
+        apiKey: openAIKey,
+      })
+    )
+    const openAIResponse = await openAI.createChatCompletion({
```

Comment:
I suggest to create a new service (under `/server/services`) for methods related to generative AI

Reply:
> I suggest to create a new service (under `/server/services`) for methods related to generative AI

created the service here https://github.com/kixlab/kuiz/pull/81/commits/03082054d8bbd6da2b4927ab4910e49ad74f22d3 :)